tools/packaging/static-build/virtiofsd/build-static-virtiofsd.sh

```diff
@@ -40,4 +40,43 @@ pull_virtiofsd_released_binary() {
     popd
 }
 
-pull_virtiofsd_released_binary
+init_env() {
+   case ${ARCH} in
```

Comment:
Please, add `x86_64` as part of the witch as well.  And, as aarch64, it should be using musl.

Reply:
Thanks @fidencio , I will fix it based on all above comments.